app/Http/Controllers/MessageController.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Message;
+use Carbon\Carbon;
+use DateTime;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use Validator;
+
+class MessageController extends Controller
+{
+    protected function validator(array $data)
+    {
+        return Validator::make($data, [
+            'description' => 'required|max:255',
```

Comment:
is description the correct variable name? i think it is better with message or content right?